webrtc/RTCPeerConnection-addTrack.https.html

```diff
@@ -81,21 +137,6 @@
 
       assert_equals(sender.track, track,
         `Expect sender's track to be the added track`);
-
-      const transceivers = pc.getTransceivers();
-      assert_equals(transceivers.length, 1,
-        'Expect only one transceiver with sender added');
-
-      const [transceiver] = transceivers;
-      assert_equals(transceiver.sender, sender);
-
-      assert_array_equals([sender], pc.getSenders(),
-        'Expect only one sender with given track added');
-
-      const { receiver } = transceiver;
-      assert_equals(receiver.track.kind, 'audio');
-      assert_array_equals([transceiver.receiver], pc.getReceivers(),
-        'Expect only one receiver associated with transceiver added');
     });
   }, 'addTrack with single track argument and no mediaStream should succeed');
 
```

Comment:
Optional: There are other tests that could use the same setup function as above, if you care to drive-by.

Reply:
i know. this is just a strawman thing :-)